File: Modul-3/Soal-2/PRAK302-2310817110013-RANDY_FEBRIAN.c
#include <stdio.h>

int main (){
    int nilai;
    for (int i = 0; i < 5; i++){
        scanf("%d", &nilai);
        if(nilai >= 80){
            printf("A\n");
        } else if(nilai >= 70 && nilai < 80){
            printf("B\n");
        } else if (nilai >= 60 && nilai < 70){
            printf("C\n");
        } else if(nilai >= 50 && nilai < 60){
            printf("D\n");
        } else{
            printf("E\n");
        }        
    }
    return 0;
}

File: Modul-3/Soal-2/PRAK302-2310817110013-RANDY_FEBRIAN.py
for i in range(5):
    nilai = int(input())
    if(nilai >= 80):
        print("A")
    elif 70 <= nilai < 80:
        print("B")
    elif 60 <= nilai < 70:
        print("C")
    elif 50 <= nilai < 60:
        print("D")
    else:
        print("E")